### PR TITLE
module: tables

### PR DESCRIPTION
**Pull request by PraepechPrAe**

### CODE REVIEW

File: src/tables/tables.controller.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    description: 'Create table',
+  })
+  @Post()
+  createTable(@Body() tables: TablesDto) {
+    return this.tablesService.createTable(tables._id);
+  }
+
+  @ApiResponse({
+    status: HttpStatus.OK,
+    description: 'Get tables',
```
Comment (ishiasaka) [inline src/tables/tables.controller.ts:23]: Please define response type (`TableSchema`) and isArray

**Review by ishiasaka — CHANGES_REQUESTED** [review]

File: src/tables/tables.controller.ts (hunk 1 above)
Comment (PraepechPrAe) [reply src/tables/tables.controller.ts:23]: Doneeee

**Review by ishiasaka — APPROVED** [review]
Great Job!

**Comment by PraepechPrAe** [comment]
tables

**Pull request merged**